### PR TITLE
Tests for new endpoints

### PR DESCRIPTION
**Pull request by Nataliia-Popkova**
## PR Description

warehouse-use-assignment
service-points
push-notification-provides

## Checklist
- [ x ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: tests/api/suite/bapi/push_notification_endpoints/push_notification_provides/negative.robot [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    When I send a PATCH request:    /push-notification-providers/${push_notification_provider_id_2}  {"data": {"type": "push-notification-providers","attributes": {"name": "Provider1 ${random}"}}}
+    Then Response status code should be:    400
+    And Response should return error code:    5008
+    And Response should return error message:    A push notification provider with the same name already exists.
+    [Teardown]     Run Keywords    I send a DELETE request:    /push-notification-providers/${push_notification_provider_id}
+    ...    AND    I send a DELETE request:    /push-notification-providers/${push_notification_provider_id_2}
+
+Delete_push_notification_provider_with_not_exist_id
+    Then I send a DELETE request:    /push-notification-providers/invalid
+    Then Response status code should be:    404
```
Comment (AntonShopin) [inline tests/api/suite/bapi/push_notification_endpoints/push_notification_provides/negative.robot:133]: don't we need to add authorization in headers for this test?

File: tests/api/suite/bapi/service_points/service_points/negative.robot [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    [Tags]    skip-due-to-issue
+    [Setup]    Run Keywords    I get access token by user credentials:   ${zed_admin.email}
+    ...    AND    I set Headers:    Content-Type=text/plain   Authorization=Bearer ${token}
+    When I send a POST request:    /service-points   {"name": "Invalid Content Type", "key": "invalid_content_type", "isActive": "true", "stores": ["DE", "AT"]}
+    Then Response status code should be:    400
+    And Response should return error code:    5401
+    And Response should return error message:    Wrong request body.
+
+Create_Service_Point_With_Invalid_Token
+    [Setup]    I set Headers:    Authorization=InvalidToken
```
Comment (AntonShopin) [inline tests/api/suite/bapi/service_points/service_points/negative.robot:91]: Maybe `Authorization=Bearer InvalidToken`?

File: tests/ui/e2e/b2c.robot [hunk 3]
```diff
@@ -148,6 +148,7 @@ Catalog_Actions
     Yves: 1st product card in catalog (not)contains:     Add to Cart    false
     Yves: perform search by:    002
     Yves: 1st product card in catalog (not)contains:      Add to Cart    true
+    Set Timeout    2s
```
Comment (AntonShopin) [inline tests/ui/e2e/b2c.robot:151]: please remove it

**Review by AntonShopin — CHANGES_REQUESTED** [review]

File: resources/steps/warehouse_user_assigment_steps.robot [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    [Documentation]    This keyword creates new warehouse user assigment in the DB table `spy_warehouse_user_assignment`. 
+        ...    *Example:*
+        ...
+        ...    ``Create_warehouse_user_assigment:   ${warehouse[0].warehouse_uuid}    ${warehous_user[0].user_uuid}    false``
+        ...
+    [Arguments]    ${warehouse_uuid}    ${user_uuid}    ${isActive}
+    ${idWarehouseUserAssignment}=    Get next id from table    spy_warehouse_user_assignment    id_warehouse_user_assignment
+    Connect to Spryker DB
+    IF    '${db_engine}' == 'pymysql'
+        Execute Sql String    insert ignore into spy_warehouse_user_assignment (uuid, fk_warehouse, user_uuid, is_active) value ('${warehouse_uuid}','9', '${user_uuid}', ${isActive});
```
[the remark this answers is not in the document]
Comment (Nataliia-Popkova) [reply resources/steps/warehouse_user_assigment_steps.robot:19]: fixed

File: tests/api/suite/bapi/push_notification_endpoints/push_notification_provides/negative.robot (hunk 1 above)
Comment (Nataliia-Popkova) [reply tests/api/suite/bapi/push_notification_endpoints/push_notification_provides/negative.robot]: fixed

**Pull request closed without merging**